core/src/components/modal/gestures/sheet.ts

```diff
@@ -113,6 +140,7 @@ export const createSheetGesture = (
      * allow for scrolling on the content.
      */
     const content = (detail.event.target! as HTMLElement).closest('ion-content');
+    currentBreakpoint = getCurrentBreakpoint()!;
```

Comment:
In all instances current breakpoint should be defined. This appears to be a side-effect of the types set on the current breakpoint. We can likely add a check before creating the sheet gesture to avoid the modifier though.